src/tests/simple_tests/testdata/log_precompile/entry.asm

```diff
@@ -0,0 +1,37 @@
+        .text
+        .file	"Test_26"
+        .rodata.cst32
+        .p2align	5
+        .text
+        .globl	__entry
+    __entry:
+    .main:
+        add 10000, r0, r1
+        add 1000, r0, r10
+        ; put 2 in r2 (sha)
+        add 2, r0, r2
+        ; precompile call - address 0x1, AuxData (additional costs - 0 for now)
+        log.precompile r2, r3, r4
+
+        near_call r1, @inner, @handler
+        ; We should never get here - as the near_call should panic due to out of gas.
+        ret.panic r0
+    inner:
+        to_l1 r0, r1
+        ; put 2 in r2 (sha)
+        add 2, r0, r2
+
+        ; Add extra 16k cost (more than this near call has)
+        add 1, r0, r3
+        shl.s 14, r3, r3
+        shl.s 18, r3, r3
+        add 1, r3, r3
+        shl.s 14, r3, r3
+
+        ; precompile call - address 0x1, AuxData (additional costs - 16k for now)
+        log.precompile r2, r3, r4
```

Comment:
it seems like aux abi is 2^14 * (2^32 + 1) => 2^14 pubdata bytes and 0 ergs. Is it the intended behavior or I do not read it correctly?

Reply:
((1<<32) + 1 )<<14.
So both 'u32' fields should have 2^14.

Reply:
agree, yes